options.go

```diff
@@ -176,6 +180,38 @@ func (o *Options) Validate() error {
 		}
 	}
 
+	if o.SkipJwtBearerTokens {
+		// If we are using an oidc provider, go ahead and add that provider to the list
+		if o.oidcVerifier != nil {
+			o.jwtBearerVerifiers = append(o.jwtBearerVerifiers, o.oidcVerifier)
+		}
+		// Configure extra issuers
+		if len(o.ExtraJwtIssuers) > 0 {
+			msgs = parseJwtIssuers(o, msgs)
+			for _, pair := range o.jwtIssuers {
+				issuer, audience := pair[0], pair[1]
```

Comment:
Might be nice to break most of this logic into a new method, `newJwtVerifier` perhaps? WDYT? Means you can early return if you construct the OIDC verifier without error and means we have less context embedded in 4 levels of indentation that isn't important to the flow

Reply:
Addeed a JwtIssuer struct and refactored code above